fvtest/tril/tril/ilgen.hpp

```diff
@@ -72,7 +72,7 @@ class OpCodeTable : public TR::ILOpCode {
             std::string opcodeAndSrcType = name;
             TR::DataType resType = TR::NoType;
 
-            std::size_t pos = name.find("To");
+            std::size_t pos = name.find('_');
```

Comment:
The `To` in the opcode name in the example on L71 should be changed to a `_`, presumably.

Reply:
Yes, please change the example too.